site/content/installation/integrations/nic-saas-console.md

```diff
@@ -0,0 +1,135 @@
+---
+doctypes:
+   - concept
+title: Connect NGINX Ingress Controller to NGINX One Console
+toc: true
+weight: 1800
+---
+
+This document explains how to connect F5 NGINX Ingress Controller to NGINX One Console using NGINX Agent.
+
+## Overview of NGINX One SaaS Console and Agent
+
+NGINX One SaaS Console is a cloud-based management platform that provides visibility and control over your NGINX deployments. NGINX Agent enables communication between NGINX Ingress Controller and the NGINX One SaaS Console.
```

Comment:
We use `NGINX One Console` throughout our docs, especially in https://docs.nginx.com/nginx-one

I realize that it's a cloud product, but I don't see SaaS referenced anywhere else.

Reply:
Correct -- we do not use "SaaS" in the official product name.